### ISSUE TITLE
Can this force disable Dolby Vision Globally?

### ISSUE THREAD

**Issue by SyCoREAPER** [Maassoft/ColorControl#404]
Something is wrong with my set 65 C1 (non-EVO) that I didn't notice until way past the warranty. 

Dolby Vision causes blooming or "hot spots", especially noticeable in the bottom corners when it should be completely black if I go above 48 Brightness. I don't know why and no combination of settings or configurations fixes it. Vanilla HDR and SDR are fine. It's the isolated sections of the screen. If say I'm watching a space movie,most of the picture will be black as it should be but somewhere in the middle there's also a section that hot and breaks the immersion as space is glowing from this defect. 

A true reset didn't fix it when I accidentally hit 'In-Stop' 🤦‍♂️ and left everything factory and tried going above 48 or with my settings. Doesn't matter the input or if it's built-in apps. 

I've (fortunately) stopped updating and am rooted but don't see anything in terms of Homebrew to disable it. This seems like the only thing if anything that might work.

**Comment by Maassoft** [comment]
If your tv is rooted, you probably know how to connect with a terminal (SSH) to the tv and execute commands?
If so, the following command will most likely disable Dolby Vision on your tv:

`luna-send -n 1 -f "luna://com.webos.service.config/setConfigs" '{ "configs": { "tv.model.supportDolbyVisionHDR": false } }'`

But I haven't tested this myself and you execute this at your own risk.

**Comment by SyCoREAPER** [comment]
Just curious and not being a wise ass but where did you get that string from if you haven't tested it?

While your help and input is appreciated, I'm not exactly enticed by the possibility of a brick

**Comment by Maassoft** [comment]
From decrypted firmware files. This gives me access to all the files of the filesystem of a LG tv. Also, I had my C9 rooted once and I've extracted some config .json files which contain lists of all the supported features.
For instance, "tv.model.motionProMode" is such a setting that can even be manipulated via the web service (via a hack) which allowed me to enable "OLED Motion Pro" on my C9 3 years ago.
The setting "tv.model.supportDolbyVisionHDR" is just the most likely one to disable/enable Dolby Vision. You can always revert back by specifying 'true', but yeah, not getting a brick is not guaranteed.
There's also a config "tv.config.supportDolbyHDRContents", maybe that one is a bit safer because when set to 'false' it will probably only report that it doesn't support Dolby Vision and therefore not use it for built-in apps. But I haven't tested any of these, so again, you're on your own here.

**Comment by SyCoREAPER** [comment]
Nevertheless I appreciate it. I'm going to have to sit on this a bit. 

I don't know enough about WebOS but one thing has me worried is that if I change the DV value mentioned is that the Widevine L1 equivalent of WebOS (if there is a such thing) will report a mismatch and put me into L3 (again if that exists in WebOS, I'm an Android guy and maybe thinking too much into it). 
I'm going to really have to sit on this. AV1, which I was shilling and am actually against as of last week, no middleman (AVR), no extra equipment, but a potential for a brick (and maybe referring to lower resolution). Or still take the FireCube v3 out back and shoot it and run my newly downgraded Shield Pro which is just as snappy now just no AV1 but has an AVR as a man in the middle. Nothing is ever simple. Nevertheless, thank you for your help in what ended up going OT from your software. 

Welp, in the meantime, I've played with your wonderful app enough that I now fully understand what it's intended to do and will get what I was going to do in Powershell for each game done in a lot less painless way.

**Comment by SyCoREAPER** [comment]
Disregard. 

Anyone else reading this. This is a lost cause. I figured out the SSH issue... what an idiot I am. Mowed the grass and 24 hours straight, it's Port 22 for ssh and port 23 for telnet.

That said, I pressed my luck and installed CLITools and WebOS Dev Manager. Got basically nowhere. Looks like if one hadn't set up SSH in the past with a new PW and Passphrase, they are locked in some emulated state, the password rotates. I created the accounts on the TV and even while in CLI/Dev Tools I couldn't connect with third-party clients.

The dream is dead. Was able to re-root, so I guess that's a win. But this thread, which went very OT, has run its course.
